lecture_2/homework/app/controllers/clans/warriors_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Clans
+  class WarriorsController < ApplicationController
+    def show
+      render json: warrior.as_json(include: 'clan', include: 'defensible')
```

Comment:
Jak użyjesz `as_json` na obiekcie to ActiveModelSerialziers nie użyje serializera `json_api`, który wybrałaś tylko `ActiveModel::Serializer::Null`. Ten serializer nie robi nic innego jak wypisanie wszystkich atrybutów obiektu. Ta linia powinna wyglądać jakoś tak:
```ruby
render json: warrior, include: ['clan', 'defensible']
```